README.md

```diff
@@ -116,7 +116,7 @@ Set `backup_files_storage_type = "s3"` if you have an AWS s3 bucket somewhere wi
 
 ### Using Local Files
 
-Set `backup_files_storage_type = "local"` if the save files are on your local host PC. Please note the warning below about the REQUIRED file structure of the bucket. Terraform and user_data are opinionated in how they retrieve and place these files, so you must adhere to this structure.
+Set `backup_files_storage_type = "local"` if the save files are on your local host PC. Please note the warning below about the REQUIRED file structure of the bucket [e.g. of file structure required: C:/Users/<username>/Desktop/palworld_aws/Saved/SaveGames/0/AC585CF94526439EA4A8F5AC615E5CDB]. Do not include the /Players in the file structure, stop at the hash after /0/. Additionally if using WINDOWS saves during migration, use EpicApp=PalServer -nosteam instead of EpicApp=PalServer. Terraform and user_data are opinionated in how they retrieve and place these files, so you must adhere to this structure.
```

Comment:
This module ALWAYS applies the EpicApp-PalServer option. I have not configured a bool input for epic_app = true or false. If this is something that is desired along with the -nosteam argument then I can modify the module. Are you willing to open a feature request issue and request this?